src/main/java/org/terasology/logic/health/HealthAuthoritySystem.java

```diff
@@ -21,20 +21,33 @@
 import gnu.trove.list.TIntList;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.terasology.audio.StaticSound;
+import org.terasology.audio.events.PlaySoundEvent;
+import org.terasology.audio.events.PlaySoundForOwnerEvent;
+import org.terasology.engine.Time;
 import org.terasology.entitySystem.entity.EntityManager;
 import org.terasology.entitySystem.entity.EntityRef;
 import org.terasology.entitySystem.event.ReceiveEvent;
+import org.terasology.entitySystem.prefab.Prefab;
 import org.terasology.entitySystem.systems.BaseComponentSystem;
 import org.terasology.entitySystem.systems.RegisterMode;
 import org.terasology.entitySystem.systems.RegisterSystem;
 import org.terasology.entitySystem.systems.UpdateSubscriberSystem;
-import org.terasology.logic.health.event.BeforeHealEvent;
-import org.terasology.logic.health.event.DoHealEvent;
-import org.terasology.logic.health.event.FullHealthEvent;
-import org.terasology.logic.health.event.OnHealedEvent;
+import org.terasology.logic.characters.CharacterMovementComponent;
+import org.terasology.logic.characters.CharacterSoundComponent;
+import org.terasology.logic.characters.CharacterSoundSystem;
+import org.terasology.logic.characters.MovementMode;
+import org.terasology.logic.characters.events.AttackEvent;
+import org.terasology.logic.characters.events.HorizontalCollisionEvent;
+import org.terasology.logic.characters.events.VerticalCollisionEvent;
+import org.terasology.logic.health.event.*;
```

Comment:
Avoid star imports.
If IntelliJ is auto collapsing them then you can change the settings to increase the number of classes before it collapses to some high number like 999